app/src/main/res/color/onboarding_illustration_color_state.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit - license
And same question as above - why do we need both selected and checked?

Reply:
This selector is used for both the radio button color as well as the illustration border, this is why I used both states. Separated into 2 files to clear any confusion